src/Bau/Arguments.cs

```diff
@@ -14,17 +14,20 @@ public class Arguments
     {
         private readonly ReadOnlyCollection<string> tasks;
         private readonly LogLevel logLevel;
+        private readonly TaskListWriter taskListWriter;

```

Comment:
This is inappropriate here. `Arguments` is designed to encapsulate the command line arguments. It should have new properties added to represent the new command line options but it should not have the added responsibility of creating a task list writer.